File: FrontEnd/src/app/features/reservations/manifest/classes/view-models/list/manifest-crew-vm.ts
import { ManifestNationalityVM } from './manifest-nationality-vm'
import { SimpleEntity } from 'src/app/shared/classes/simple-entity'

export interface ManifestCrewVM {

    id: number
    lastname: string
    firstname: string
    birthdate: string
    phones: string
    gender: SimpleEntity
    nationality: ManifestNationalityVM
    port: SimpleEntity
    specialty: SimpleEntity

}

File: API/Features/Reservations/Manifest/ViewModels/List/ManifestCrewVM.cs
using API.Infrastructure.Classes;

namespace API.Features.Reservations.Manifest {

    public class ManifestCrewVM {

        public int Id { get; set; }
        public string Lastname { get; set; }
        public string Firstname { get; set; }
        public SimpleEntity Gender { get; set; }
        public SimpleEntity Specialty { get; set; }
        public ManifestNationalityVM Nationality { get; set; }
        public string Birthdate { get; set; }

    }

}